### PR TITLE
fix compiler warnings

### PR DESCRIPTION
**Pull request by TorreZuk**

### CODE REVIEW

**Review by AlexBrownAMD — APPROVED** [review]

File: Tensile/Source/lib/include/Tensile/PropertyMatching.hpp
Comment (nakajee) [reply Tensile/Source/lib/include/Tensile/PropertyMatching.hpp:698]: Can we use std::cout instead of using printf?

Comment (TorreZuk) [reply Tensile/Source/lib/include/Tensile/PropertyMatching.hpp:698]: @nakajee you can push a commit to this PR with cout as I just want the warning fixed, you can use whatever Tensile team prefers (these seem to be size_t members for the warnings you get in CI, see ).   Is there still no run of your own windows CI run directly on Tensile?

Comment (nakajee) [reply Tensile/Source/lib/include/Tensile/PropertyMatching.hpp]: @TorreZuk 
I pushed cout version.
Even though we changed ld to lld, I can still see the opposite version of warning as follows.
===

/var/jenkins_home/workspace/clang_precheckin_Tensile_PR-1623/ZwdxdW0y1/tensile/Tensile/Source/lib/include/Tensile/PropertyMatching.hpp:698:64: warning: format specifies type 'long long' but the argument has type '__gnu_cxx::__alloc_traits<std::allocator<long>, long>::value_type' (aka 'long') [-Wformat]

                        printf("[%lld,%lld,%lld]\n", e.key[0], e.key[1], e.key[2]);

                                      ~~~~                     ^~~~~~~~

                                      %ld
===

Let's see if using cout can fix the warning or not.

Comment (nakajee) [reply Tensile/Source/lib/include/Tensile/PropertyMatching.hpp]: By the way, probably, this is not just win compiler issue.

**Review by TorreZuk — COMMENTED** [review]
Very good, windows clang is slightly different version to linux clang, my change built rocblas without windows warnings. But yes if you are happy with that format should fix both, it's debug output so hopefully not parsed.

**Comment by TorreZuk** [comment]
PS - someone on tensile side should probably see if the type instantiation should be the same on windows and linux (i.e. long or long long)).  Unrelated to fixing the warning but could be a problem down the road.

**Comment by TorreZuk** [comment]
@nakajee thanks for taking this over, I assume I can leave it to you to merge as you will know what failures are the norm and can be ignored?

**Comment by nakajee** [comment]
> @nakajee thanks for taking this over, I assume I can leave it to you to merge as you will know what failures are the norm and can be ignored?

I can handle merge.
I am still waiting for CI tests.
Fails on extended 906 is expected (will be fixed with #1617).
Not sure why 908 taking so long time.

**Pull request merged**
